src/containerapp/azext_containerapp/commands.py

```diff
@@ -88,6 +88,17 @@ def load_command_table(self, _):
         g.custom_command('set', 'create_or_update_storage', supports_no_wait=True, exception_handler=ex_handler_factory())
         g.custom_command('remove', 'remove_storage', supports_no_wait=True, confirmation=True, exception_handler=ex_handler_factory())
 
+    with self.command_group('containerapp service') as g:
+        g.custom_command('list', 'list_all_services')
+
+    with self.command_group('containerapp service redis') as g:
+        g.custom_command('create', 'create_redis_service', supports_no_wait=True)
+        g.custom_command('delete', 'delete_redis_service', confirmation=True, supports_no_wait=True)
```

Comment:
'as containerapp service redis delete' and 'az containerapp service postgres delete' commands need tests cover as they are new commands.